docs/docs/rad-lab-ui/ui_installation/frontend.mdx

```diff
@@ -60,19 +58,50 @@ Add your Project ID here to have the documentation be be customized to your Goog
 
 <ProjectIdInput />
 
-## Add Firebase
+# Add Firebase
+
+:::warning
+If you've never used Firebase in your Google Cloud organization before, please execute [First Time Use](#first-time-use).  If you already have Firebase projects running in your Google Cloud organization or you have in the past, please execute [Returning Firebase User](#returning-firebase-user).
 
-:::important First time
-If you have never used Firebase in your Google Cloud organization, you need to manually add it to the Google Cloud project created in the first step. This is necessary as you have to accept the Terms & Conditions first.
+If you're not sure, simply execute the steps as described in [First Time Use](#first-time-use).
```

Comment:
If you're not sure, simply follow the steps as described in [First Time Use](#first-time-use).